src/server/db.js

```diff
@@ -0,0 +1,60 @@
+var mysql     =    require('mysql');
+
+// URL of the MySQL server. We expect that the DB is listening on the
+// default port (3306).
+//
+const DB_HOST = 'heavenly';
+const DB_USER = 'nyanCat';
+const DB_PASSWD = 'P!b2BtW8AarP'; // Must match value in createSchema.sql
+const DB_SCHEMA = 'nyanPortal';
+
+var pool      =    mysql.createPool({
+    connectionLimit : 100, //important
+    host     : DB_HOST,
+    user     : DB_USER,
+    password : DB_PASSWD,
+    database : DB_SCHEMA,
+    debug    : false
+});
+
+/* This wrapper converts the callback-based MySQL entry points to Promises
+ * that resolve to DB rows if the query was completed successfully.
+ */
+ 
+export function dbQuery(queryStr) {
```

Comment:
This is a nitpick, but I'd just call this function `query` since the module is already called `db`.